inference-engine/src/mkldnn_plugin/nodes/mkldnn_split_node.cpp

```diff
@@ -362,9 +335,16 @@ void MKLDNNSplitNode::initOptimalPrimitiveDescriptor() {
 }
 
 void MKLDNNSplitNode::selectOptimalPrimitiveDescriptor() {
-    if (implPriorities.size() > 0 && implPriorities[0] == impl_desc_type::ref) {
-        selectPrimitiveDescriptorByIndex(0);
-        return;
+    if (!implPriorities.empty() && implPriorities[0] == impl_desc_type::ref) {
```

Comment:
Could you please add the comment why we need special case here

Reply:
Done